packages/frontend/core/src/hooks/affine/use-app-setting-helper.ts

```diff
@@ -0,0 +1,23 @@
+import { useAtom } from 'jotai';
+import { useCallback, useMemo } from 'react';
+
+import { type AppSetting, appSettingAtom } from '../../atoms/settings';
+
+export function useAppSettingHelper() {
+  const [appSettings, setAppSettings] = useAtom(appSettingAtom);
+
+  const updateSettings = useCallback(
+    <K extends keyof AppSetting>(key: K, value: AppSetting[K]) => {
+      setAppSettings(prevSettings => ({ ...prevSettings, [key]: value }));
+    },
+    [setAppSettings]
+  );
+
+  return useMemo(
```

Comment:
This memo is unnecessary

Reply:
This is added because how the return value of `useAppSettingHelper` is being used as an `useEffect` dep in `registerAffineSettingsCommands`. Similar to other useMemo usage in this PR.

I guess the proper way of doing so is to passing individual values to `registerAffineSettingsCommands` instead?